warehouse/manage/views.py

```diff
@@ -1578,6 +1673,13 @@ def delete_project_role(project, request):
             request.session.flash("Cannot remove yourself as Owner", queue="error")
         else:
             request.db.delete(role)
+            role_invite = (
+                request.db.query(RoleInvitation)
+                .filter(RoleInvitation.project == project)
+                .filter(RoleInvitation.user == role.user)
+                .one()
+            )
```

Comment:
Same thing here, use the ID instead.

Reply:
As discussed, we decided to leave a minimal footprint in the role invitation table, thus marking this as resolved.